rootfs/scheduler/__init__.py

```diff
@@ -525,6 +525,12 @@ def run(self, namespace, name, image, entrypoint, command, **kwargs):
             if waited == timeout:
                 raise KubeException('Timed out (20 mins) while running')
 
+            # check if it is possible to get logs
+            state = self.pod_state(self.get_pod(namespace, name).json())
+            # Only get states between up and down
+            if state < PodState.up and state > PodState.down:
+                return exit_code, 'Could not get logs. Pod is in state {}'.format(str(state))
```

Comment:
i think we should get the logs even if the pod went into error state because it might show some info about the failure.